defs & perspectives (#71)
cherryontech/website: content/dictionary/foo.md

---
title: foo

> definition: Foo and foobar is commonly recognized placeholders for generic examples, files, or programs.
> sources: 
> - sourceurl: https://softwareengineering.stackexchange.com/questions/69788/what-is-the-history-of-the-use-of-foo-and-bar-in-source-code-examples/69807
> perspectives: 
> - meaning: Commonly used word to name variables, usually in examples that do not necessarily matter.

Reviewer: ```suggestion
definition: Foo and foobar are commonly recognized placeholders for generic examples, files, or programs. They are commonly used words to name variables, usually in examples that do not necessarily matter.
sources: 
- sourceurl: https://softwareengineering.stackexchange.com/questions/69788/what-is-the-history-of-the-use-of-foo-and-bar-in-source-code-examples/69807
perspectives: 
```
Since the added information is applicable to the entire definition and not just one role, I'd suggest adding it to the definition.
